apps/keyboard/test/unit/keyboard/user_press_manager_test.js

```diff
@@ -54,6 +56,7 @@ suite('UserPressManager', function() {
     domObjMap = new WeakMap();
 
     app = {
+      settingsPromiseManager: new SettingsPromiseManager(),
```

Comment:
This has to be a stub instead of the real thing, or we will be calling mozSettings API for real.